eval/mt_bench/components.py

```diff
@@ -188,14 +188,16 @@ def shutdown_vllm(process: subprocess.Popen, timeout: int = 20):
         scores[model_path] = overall_score
 
     with open(output_path, "w", encoding="utf-8") as f:
-        json.dump(all_mt_bench_data, f, indent=4)
+        for data in all_mt_bench_data:
+            json.dump(data, f)
+            f.write("\n")
```

Comment:
Can't this be a JSON instead of JSONL? It's a JSON Array in the end. A file containing `[{...},{...}]` is a valid JSON, isn't it?

I still don't understand the parsing issue, if it was a JSON in the first place...

Reply:
You are right. I re-evaluated this step just using json instead of jsonl and it seems to work as expected. 🙃 Will revert this change.